src/composeappmanager.cc

```diff
@@ -506,8 +525,10 @@ data::InstallationResult ComposeAppManager::finalizeInstall(const Uptane::Target
 
       app_engine_->prune(app_shortlist);
     }
+    if (!newly_enabled_apps_msg.empty()) {
+      ir.description += newly_enabled_apps_msg;
+    }
   }
-
```

Comment:
nitpick: line removal unrelated to change.

Reply:
fixed